src/librustc/dep_graph/dep_node.rs

```diff
@@ -458,6 +458,7 @@ define_dep_nodes!( <'tcx>
     [eval_always] PrivacyAccessLevels(CrateNum),
     [eval_always] CheckPrivateInPublic(CrateNum),
     [eval_always] Analysis(CrateNum),
+    [eval_always] HirMap(CrateNum),
```

Comment:
Probably need a true `eval_always` query type, since these are actually eval if HIR changes, and we don't know if the HIR changes until after the HIR is hashed, which happens in the HirMap query.

Reply:
This refers to https://github.com/rust-lang/rust/pull/59091, right?

Reply:
Yeah.